GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -112,6 +112,14 @@ protected override void OnRuntimeLoad(EventArgs e)
                     {
                         string text = Clipboard.GetText(TextDataFormat.Text) ?? string.Empty;
 
+                        // Remove git clone prefix, if any.
+                        // This could be extended in future with a regex.
+                        const string gitCloneText = "git clone";
+                        if (text.StartsWith(gitCloneText, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I don't believe git commands are case insensitive. Whilst you can call the actual git command as "GIT" on Windows, git commands are all in the lower case:
![image](https://user-images.githubusercontent.com/4403806/112627187-5ef76c00-8e85-11eb-8e16-877d33805f99.png)

Reply:
Thank you for the feedback; I decided to use a regex to attempt to detect the URL and then allow the user to modify it prior to the clone operation.